lib/commands/run/run-flow/abstract-run-flow.js

```diff
@@ -1,17 +1,21 @@
-const nodemon = require("nodemon");
+import nodemon from "nodemon";
+import Messenger from "../../../view/messenger";
 
-class AbstractRunFlow {
+export class AbstractRunFlow {
   constructor(execConfig) {
     this.execConfig = execConfig;
   }
 
   execCommand() {
-    nodemon(this.execConfig);
+    return new Promise(() => {
+      nodemon({verbose: true, ...this.execConfig})
```

Comment:
Just looking for clarification here, changes look good to me... a little fuzzy on how this makes the `nodemon` call synchronous now, if the app wasn't waiting on this line before, why would it be now?

Reply:
there is an await  where execCommand() is getting called in the index file